packages/govuk-frontend/rollup.release.config.mjs

```diff
@@ -30,7 +29,12 @@ export default defineConfig(({ i: input }) => ({
     plugins: [
       terser({
         format: { comments: false },
-        mangle: { reserved: Object.keys(GOVUKFrontend) },
+        mangle: {
+          keep_classnames: true,
```

Comment:
Do the [docs suggest we can set `minify: { keep_classnames: true }`](https://terser.org/docs/api-reference/#minify-options) then we don't need this line?

>If the top level minify option `keep_classnames` is undefined it will be overridden with the value of the top level minify option `keep_fnames`.

Might not need both, but happy to keep them for clarity

Reply:
Ohh, wasn't aware of the `minify` option. Looks like there's a subtle difference between that option and the mangling one:

`mangle`:
> Pass true to not mangle function names.

`minify` (emphasis mine):
> pass true to prevent **discarding or** mangling of class names

Looks like enabling through the `minify` option would also enable the corresponding `keep_...` [options from `compress`](https://terser.org/docs/options/#compress-options). I think we'd still want terser to remove any unused classes and functions, just not mangle any functions or classnames it keeps, so configuring what we keep through `mangle`

> > If the top level minify option `keep_classnames` is undefined it will be overridden with the value of the top level minify option `keep_fnames`.
> 
> Might not need both, but happy to keep them for clarity

Doesn't seem that the same apply for `mangle` options, just for `minify`. It's more explicit when reading the config to keep both in any case 😊

Reply:
Fair enough, thanks for checking though!